flink-libraries/flink-table/src/main/scala/org/apache/flink/table/runtime/aggregate/AggregateUtil.scala

```diff
@@ -785,7 +785,7 @@ object AggregateUtil {
     (propPos._1, propPos._2)
   }
 
-  private def transformToAggregateFunctions(
```

Comment:
It is not to much logic...and the overall "createTimeBoundedProcessingTimeOverWindow" is not bigger than the other method for unbound.
...as I said...it is the same...so as you prefer